Dockerfile.travis

```diff
@@ -0,0 +1,88 @@
+FROM ubuntu:bionic AS pybm-travis-bionic
+
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    software-properties-common
+
+RUN dpkg --add-architecture i386
+
+RUN add-apt-repository ppa:deadsnakes/ppa
+
+RUN apt-get update
+
+RUN apt-get -y install sudo
+
+# travis xenial bionic
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python-setuptools libssl-dev libpq-dev python-prctl python-dev \
+    python-dev python-virtualenv python-pip virtualenv
+
+# travis focal
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python-setuptools libpcap-dev libpq-dev python-prctl python-dev \
+    libpcap-dev python3-virtualenv python3-pip virtualenv
+
+# dpkg
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python-minimal python-setuptools python-all python openssl libssl-dev \
+    dh-apparmor debhelper dh-python python-msgpack python-qt4 python-stdeb \
+    python-all-dev python-crypto python-pycryptopp python-psutil \
+    fakeroot python-pytest
+
+# Code quality
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    pylint python-pycodestyle python3-pycodestyle pycodestyle python-flake8 \
+    python3-flake8 flake8 python-pyflakes python3-pyflakes pyflakes pyflakes3 \
+    curl
+
+# Wine32
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python python-pip wget wine-stable winetricks mingw-w64 wine32 wine64 xvfb
+
+# Wine64
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python python-pip wget wine-stable winetricks mingw-w64 wine32 wine64 xvfb
+
+# Buildbot
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python3-dev libffi-dev python3-setuptools \
+    python3-pip
+
+# python 3.7
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    python3.7 python3.7-dev
+
+# .travis.yml
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    build-essential libcap-dev tor
+
+# locales
+RUN apt-get install -yq --no-install-suggests --no-install-recommends \
+    language-pack-en
+
+# cleanup
+RUN rm -rf /var/lib/apt/lists/*
+
+RUN adduser builder
+
+RUN echo 'builder ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
+
+# travis2bash
+RUN wget -O /usr/local/bin/travis2bash.sh https://git.bitmessage.org/Bitmessage/buildbot-scripts/raw/branch/master/travis2bash.sh
+RUN chmod +x /usr/local/bin/travis2bash.sh
+
+COPY . /home/builder/src
+RUN chown -R builder.builder /home/builder/src
```

Comment:
Strange. `man chown` says `chown [OPTION]... [OWNER][:[GROUP]] FILE...`

Reply:
I think OWNER.GROUP is a GNU non-POSIX feature, maybe it doesn't work on BSD, I've been using it on linux since the '90s. But I can change it, no big deal.